src/org/opensolaris/opengrok/web/suggester/provider/service/SuggesterServiceFactory.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opensolaris.opengrok.web.suggester.provider.service;
+
+import org.opensolaris.opengrok.web.suggester.provider.service.impl.SuggesterServiceImpl;
+
+public class SuggesterServiceFactory {
+
+    private SuggesterServiceFactory() {
+    }
+
+    public static SuggesterService getDefault() {
```

Comment:
a javadoc wouldn't kill you :D something like what to expect from the default implementation

Reply:
I was thinking about it but I really hate obvious comments :D 

There will probably never be any other implementation of `SuggesterService` + this class will be probably deleted once the `@Inject` could be used freely